engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1875,7 +1948,7 @@ static void LogFrameBufferError(GLenum status)
                 uint32_t stream_offset = base_offset + vertex_declaration->m_Streams[i].m_Offset;
                 uint32_t sub_vector_component_count = sub_vector_bind_count > 1 ? sub_vector_bind_count : vertex_declaration->m_Streams[i].m_Size;
 
-                for (int j = 0; j < sub_vector_bind_count; ++j)
+                for (uint32_t j = 0; j < sub_vector_bind_count; ++j)
```

Comment:
Cosmetic type change. `sub_vector_bind_count` is `uint32_t` type and below functions operates with unsigned types.